### ISSUE TITLE
Render Silhouette For A Trimesh Object

### ISSUE THREAD

**Issue by adityab05** [daniilidis-group/neural_renderer#138]
Hi, I already have a trimesh object i have stored using the below command,
`mesh.export('name.obj')`

Now what I want to do is, Using weak camera perspective get the 2D silhouette rendered.
I changed the code in example1.py as seen below,
`images = renderer(vertices, faces, mode = 'silhouettes')  # [batch_size, RGB, image_size, image_size]
 image = images.detach().cpu().numpy()[0].transpose((1, 2, 0))  # [image_size, image_size, RGB]`

but I get an error as seen below,
![image](https://user-images.githubusercontent.com/56682767/180910503-937de45b-3ab2-4924-a701-ac1fe058cec6.png)

How to resolve this ?
